modules/startioBidAdapter.js

```diff
@@ -151,6 +151,15 @@ export const spec = {
   },
 
   onSetTargeting: (bid) => { },
+
+  getUserSyncs: function (syncOptions) {
+    if (syncOptions.pixelEnabled) {
+      return [{
+        type: 'image',
+        url: 'https://sync.intentiq.com/profiles_engine/ProfilesEngineServlet?at=20&mi=10&dpi=1031540622'
```

Comment:
See #13465

Reply:
to expand on this, we don't have a way to check for consent to this vendor bc we assume you would only call your own endpoint. you should pass yourself the tcf string and parse it server side before redirecting to any consented sync partners you wish